saleor/graphql/shop/mutations.py

```diff
@@ -288,3 +288,88 @@ def perform_mutation(cls, _root, info, key_type):
+
+
+class StaffNotificationRecipientInput(graphene.InputObjectType):
+    user = graphene.ID(required=False, description="The ID of the user to notify.")
+    staff_email = graphene.String(
+        required=False, description="Email to notify instead of given staff member."
+    )
+    active = graphene.Boolean(
+        required=False, description="Is staff notification recipient active."
+    )
+
+
+class StaffNotificationRecipientCreate(ModelMutation):
+    class Arguments:
+        input = StaffNotificationRecipientInput(
+            required=True, description="Fields required to update a product."
+        )
+
+    class Meta:
+        description = "Creates a new staff notification recipient."
+        model = account_models.StaffNotificationRecipient
+        permissions = ("site.manage_settings",)
+        error_type_class = ShopError
+        error_type_field = "shop_errors"
+
+    @classmethod
+    def clean_input(cls, info, instance, data):
+        cleaned_input = super().clean_input(info, instance, data)
+        staff_email = cleaned_input.get("staff_email")
+        user = cleaned_input.get("user")
+        if not staff_email and not user:
+            raise ValidationError(
+                {
+                    "staff_notification": ValidationError(
+                        "User or staff email is required", code=ShopErrorCode.REQUIRED
+                    )
+                }
+            )
+        if user and not user.is_staff:
+            raise ValidationError(
+                {
+                    "user": ValidationError(
+                        "User has to be staff user", code=ShopErrorCode.REQUIRED
+                    )
+                }
+            )
+        cleaned_input["site_settings"] = info.context.site.settings
+        if staff_email:
+            staff_user = account_models.User.objects.filter(email=staff_email).first()
+            if staff_user:
+                cleaned_input["user"] = staff_user
+                cleaned_input["staff_email"] = None
+        return cleaned_input
+
+
+class StaffNotificationRecipientUpdate(StaffNotificationRecipientCreate):
+    class Arguments:
+        id = graphene.ID(
+            required=True, description="ID of a Staff notification recipient to update."
+        )
+        input = StaffNotificationRecipientInput(
+            required=True,
+            description="Fields required to update a staff notification recipient.",
+        )
+
+    class Meta:
+        description = "Updates a staff notification recipient."
+        model = account_models.StaffNotificationRecipient
+        permissions = ("site.manage_settings",)
+        error_type_class = ShopError
+        error_type_field = "shop_errors"
+
+
+class StaffNotificationRecipientDelete(ModelDeleteMutation):
+    class Arguments:
+        id = graphene.ID(
+            required=True, description="ID of a StaffNotification to update."
```

Comment:
Description should be updated. 
`description="ID of a StaffNotification to delete."`